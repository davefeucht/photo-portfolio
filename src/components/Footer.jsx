/*****************
* Footer component implements the footer of the application
*****************/

import React from "react";
import StyledFooter from "./styledComponents/StyledFooter.jsx";

export default function Footer (props) {

    return (
      <StyledFooter>
        Theme Copyright @2018 Dave Feucht  
      </StyledFooter>
    );
}