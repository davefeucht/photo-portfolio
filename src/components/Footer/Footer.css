.footer {
  color: #FFFFFF;
  grid-area: footer;
  background: rgba(0,0,0,.75); 
  display: grid;
  grid-template-colums: 100%;
  grid-template-rows: 100%;
  align-items: center;
  padding: 5px;
}