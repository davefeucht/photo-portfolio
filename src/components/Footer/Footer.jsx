/** ***************
* Footer component implements the footer of the application
**************** */

import './Footer.css';

import React from 'react';

function Footer() {
    return (
        <div className="footer">
            Theme Copyright @2021 Dave Feucht
        </div>
    );
}

Footer.displayName = 'Footer';

export default Footer;
