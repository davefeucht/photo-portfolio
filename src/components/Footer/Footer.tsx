/** ***************
* Footer component implements the footer of the application
**************** */

import './Footer.css';

import * as React from 'react';

const Footer = () => {
    return (
        <div className="footer">
            Theme Copyright @2023 Dave Feucht
        </div>
    );
};

Footer.displayName = 'Footer';

export default Footer;
