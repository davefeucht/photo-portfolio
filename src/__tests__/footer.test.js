import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Footer from '../components/Footer/Footer';

test('Footer displays', async () => {
    let component;
    let tree;
    component = renderer.create(
        <MemoryRouter>
            <Footer />
        </MemoryRouter>
    );
    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
