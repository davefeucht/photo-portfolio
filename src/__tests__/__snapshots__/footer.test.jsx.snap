// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer displays 1`] = `
<div
  className="footer"
>
  Theme Copyright @2021 Dave Feucht
</div>
`;
